tests/ha/multi_pod_failure/test_multi_server_pod_failure.py

```diff
@@ -871,36 +606,32 @@ def test_deletes_till_kserver_pods_fail(self):
             fail_bkt_get = rd_resp[1]
             event_di_bkt = rd_resp[2]
             fail_di_bkt = rd_resp[3]
-
             # Above four lists are expected to be empty as all pass expected
             assert_utils.assert_false(len(fail_bkt_get) or len(fail_di_bkt) or len(event_bkt_get)
                                       or len(event_di_bkt), "Expected pass in read and di check "
                                                             "operations. Found failures in READ: "
                                                             f"{fail_bkt_get} {event_bkt_get}"
                                                             f"or DI_CHECK: {fail_di_bkt} "
                                                             f"{event_di_bkt}")
-            LOGGER.info("Step 7: Successfully performed READs and verify DI on the remaining"
+            LOGGER.info("Step 4: Successfully performed READs and verify DI on the remaining"
                         " %s buckets.", remain_bkt)
-
-        LOGGER.info("Shutdown %s (K) server pods one by one and performed Deletes "
-                    "on random buckets and verified read on remaining bucket after"
-                    " each server pod down", self.kvalue)
-
-        LOGGER.info("Completed: Test to verify degraded DELETEs after each pod failure till K "
-                    "server pods down - unsafe shutdown.")
+        LOGGER.info("Step 2: %s (K) server pods shutdown one by one successfully, Verify cluster,"
+                    " services states are as expected & remaining pods status is online and "
+                    "performed Deletes on random buckets and verified read on remaining bucket "
+                    "after each server pod down", self.kvalue)
+        LOGGER.info("Completed: Test to verify DELETEs after each pod failure till K server pods"
+                    " down - unsafe shutdown.")
 
     @pytest.mark.ha
     @pytest.mark.lc
     @pytest.mark.skip(reason="Not supported in RGW yet")
```

Comment:
this cane be removed from copy object tests now as this will be supported in PI8